src/Integration.Vsix/SonarLintTagger/IssueMarker.cs

```diff
@@ -33,8 +36,11 @@ namespace SonarLint.VisualStudio.Integration.Vsix
     /// </remarks>
     internal class IssueMarker : IFilterableIssue
     {
-        public IAnalysisIssue Issue { get; }
-        public SnapshotSpan Span { get; }
+        public IAnalysisIssueVisualization IssueViz { get; }
```

Comment:
The changes to this class are hacks to minimise the churn in this PR.
This class will be removed later.